contracts/AccrualBenefactor.sol

```diff
@@ -56,10 +56,10 @@ contract AccrualBenefactor is Benefactor {
     notNullAddress(address(beneficiary))
     returns (bool)
     {
-        require(fraction > 0, "Fraction not strictly positive");
+        require(fraction > 0, "Fraction not strictly positive [AccrualBenefactor.sol:59]");
```

Comment:
Good idea to track the code line. My second thought of this would be difficult to maintain the code when there are changes to the source files afterward because it would need to update all the line number for a single change.

Maybe the alternative is to have the logging format as `error message [FileName.sol#functionName]`.